docs/source/using-executorch-export.md

```diff
@@ -1,3 +1,173 @@
-# Model Export
+# Model Export and Lowering
 
-Placeholder for top-level export documentation
\ No newline at end of file
+The section describes the process of taking a PyTorch model and converting to the runtime format used by ExecuTorch. This process is commonly known as "exporting", as it uses the PyTorch export functionality to convert a PyTorch model into a format suitable for on-device execution. This process yields a .pte file which is optimized for on-device execution using a particular backend.
+
+## Prerequisites
+
+Exporting requires the ExecuTorch python libraries to be installed, typically by running `pip install executorch`. See [Installation](getting-started.md#Installation) for more information. This process assumes you have a PyTorch model, can instantiate it from Python, and can provide example input tensors to run the model.
+
+## The Export and Lowering Process
+
+The process to export and lower a model to the .pte format typically involves the following steps:
+
+1) Select a backend to target.
+2) Prepare the PyTorch model, including inputs and shape specification.
+3) Export the model using torch.export.export.
+4) Optimize the model for the target backend using to_edge_transform_and_lower.
+5) Create the .pte file by calling to_executorch and serializing the output.
+
+## Hardware Backends
+
+ExecuTorch backends provide hardware acceleration for a specific hardware target. In order to achieve maximum performance on target hardware, ExecuTorch optimizes the model for a specific backend during the export and lowering process. This means that the resulting .pte file is specialized for the specific hardware. In order to deploy to multiple backends, such as Core ML on iOS and ARM CPU on Android, it is common to generate a dedicated .pte file for each.
+
+The choice of hardware backend is informed by the hardware that the model is intended to be deployed on. Each backend has specific hardware requires and level of model support. See the documentation for each hardware backend for more details.
+
+As part of the .pte file creation process, ExecuTorch identifies portions of the model (partitions) that are supported for the given backend. These sections are processed by the backend ahead of time to support efficient execution. Portions of the model that are not supported on the delegate, if any, are executed using the fallback ExecuTorch portable implementation. This allows for partial model acceleration when not all model operators are supported on the backend, but may have negative performance implications.
```

Comment:
"using the fallback implementation on CPU".

Reply:
it technically doesn't need to be portable ops. You can combine Core ML and fallback to XNNPACK